scripts/apps/content-filters/views/manage-filters.html

```diff
@@ -117,22 +118,32 @@ <h3 class="modal__heading" translate ng-hide="contentFilter._id">Add New Content
                 </fieldset>
             </form>
 
+            
+
             <div class="form__row">
                 <div class="sd-line-input sd-line-input--disabled">
                     <label class="sd-line-input__label">Filter Preview</label>
                     <textarea class="sd-line-input__input" id="contentFilter-preview" readonly ng-model="preview" sd-auto-height disabled></textarea>
                 </div>
             </div>
-            <div class="form__row form__row--flex">
-                <div class="sd-line-input" ng-class="{'sd-line-input--invalid': !test.article_id.$valid}">
-                    <label class="sd-line-input__label">Filter Test</label>
-                    <input class="sd-line-input__input" type="text" id="contentFilter-test" required ng-model="test.article_id" ng-change="test.test_result=null">
-                    <div class="sd-line-input__hint" translate>Use the article GUID for testing.</div>
+
+            <div class="sd-content-box sd-content-box--emphasise sd-padding--2 sd-shadow--z1">
+                <div class="form__row form__row--flex form__row--no-padding">
+                    <div class="sd-line-input" ng-class="{'sd-line-input--invalid' : test.error_response}">
+                        <label class="sd-line-input__label">Filter Test</label>
+                        <input class="sd-line-input__input" type="text" id="contentFilter-test" required ng-model="test.article_id" ng-change="test.test_result=null">
+                        <div class="sd-line-input__hint" ng-if="!test.error_response" translate>Use the article GUID for testing.</div>
+                        <div class="sd-line-input__message" ng-if="test.error_response" translate>{{test.error_response}}</div>
```

Comment:
Dynamic values can't be translated. Only use translations for static strings.

Reply:
How else should it be done?

Reply:
It should be translated in directives/controllers instead of templates. Instead of this:

```
// controller/directive
scope.label = 'test'

// template
<div translate>{{label}}</div>
```

do this:

```
// controller/directive
scope.label = gettext('test') // <- translation and string extraction here

// template (using an already translated value)
<div>{{label}}</div>
```

Reply:
The text already comes dynamically from the backend:
https://github.com/superdesk/superdesk-client-core/pull/3379/files#diff-eadb703f8679183451274506e7fcb472R138

Reply:
I thought you were translating `'Doesn\'t Match'` thing. If it text comes from the back-end, it should be translated in the backend and should come to the client already translated.